executor/aggfuncs/func_max_min.go

```diff
@@ -752,6 +754,14 @@ func (e *maxMin4Decimal) AppendFinalResult2Chunk(sctx sessionctx.Context, pr Par
 		chk.AppendNull(e.ordinal)
 		return nil
 	}
+	frac := e.args[0].GetType().Decimal
```

Comment:
The `FieldType` of the `e.args[0]` should be unchanged during execution.
So do we need to set its value every time we evaluate a new row?

Reply:
Do you mean we have to call `Round` for each row in `UpdatePartialResult` and `Slide`?